app/assets/javascripts/discourse/app/components/about-page.gjs

```diff
@@ -65,6 +68,7 @@ export default class AboutPage extends Component {
       {
         class: "site-creation-date",
         icon: "calendar-days",
+        display: true,
         text: this.siteAgeString,
       },
     ];
```

Comment:
it would be shorter and "faster" (though barely 😉) to filter those out early instead of iterating over all of them in the template:

```suggestion
    ].filter((stat) => stat.display);
```